AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -807,6 +876,9 @@ - (void)startDownload:(nullable MSReleaseDetails *)details {
           break;
         }
 
+        // Add downloaded release hash to report after installation.
```

Comment:
This comment is a little misleading as it's not guaranteed that app actually gets updated.

Reply:
Changed to:

> Store details to report new download after restart if this release is installed.

Please let me now if it's okay now.